.github/workflows/create_prerelease.yaml

```diff
@@ -0,0 +1,89 @@
+name: Prerelease
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseVersion:
+        description: 'The release version, for example 2.0.0.'
+        required: true
+
+env:
+  VERSION: ${{ inputs.releaseVersion }}
+  DOTNET_VERSION: '6.0.x'
+
+jobs:
+  BuildAndPushDockerImages:
+    strategy:
+      matrix:
+        image:
+          - amd64
+          - arm32v7
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker/setup-qemu-action@v2
+      - uses: docker/setup-buildx-action@v2
+        id: buildx
+        with:
+          install: true
+      - name: Login to docker hub
+        run: |
+          docker login -u ${{ secrets.DOCKER_LOGIN }} -p ${{ secrets.DOCKER_PASSWORD }}
+      - name: Build and push images
+        run: |
+          docker buildx build -f "./LoRaEngine/modules/LoRaWanNetworkSrvModule/Dockerfile.${{ matrix.image }}" -t $DOCKERHUB_ORGANISATION/lorawannetworksrvmodule:$VERSION-${{ matrix.image }} --output type=image,push=true "."
+          docker buildx build -f "./LoRaEngine/modules/LoRaBasicsStationModule/Dockerfile.${{ matrix.image }}" -t $DOCKERHUB_ORGANISATION/lorabasicsstationmodule:$VERSION-${{ matrix.image }} --build-arg CONTAINER_REGISTRY_ADDRESS=docker.io --output type=image,push=true "."
+      - name: Create manifests
+        run: |
+          docker manifest create $DOCKERHUB_ORGANISATION/lorawannetworksrvmodule:$VERSION $DOCKERHUB_ORGANISATION/lorawannetworksrvmodule:$VERSION-${{ matrix.image }} --amend
+          docker manifest create $DOCKERHUB_ORGANISATION/lorabasicsstationmodule:$VERSION $DOCKERHUB_ORGANISATION/lorabasicsstationmodule:$VERSION-${{ matrix.image }} --amend
+          docker manifest push $DOCKERHUB_ORGANISATION/lorabasicsstationmodule:$VERSION
+          docker manifest push $DOCKERHUB_ORGANISATION/lorawannetworksrvmodule:$VERSION
+    env:
+      DOCKERHUB_ORGANISATION: ${{ secrets.DOCKER_REPOSITORY }}  
+
+  CreateRelease:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-dotnet@v3.0.1
+        with: 
+          dotnet-version: ${{ env.DOTNET_VERSION }}
+      - name: Package Azure Function
+        run: |
+          dotnet publish LoRaEngine/LoraKeysManagerFacade -c Release
+          FUNCTION_ZIP=function-${VERSION}.zip
+          zip -r $FUNCTION_ZIP LoRaEngine/LoraKeysManagerFacade/bin/Release/net6.0
```

Comment:
nit, would be nice if we had a smart way of reusing DOTNET_VERSION to comput the net6.0 path here